### PR TITLE
Password change messages

### LINKED ISSUE #1402: Message upon setting password
Subject: Message upon setting password
Description:
Set password for first time

Results:
This message generated : "Your password has been reset and a notification was sent to your email. You can now sign in without a password."

Should above message end : "without an email link" ?

Backend version: develop-762ac12b
Frontend version: develop-762ac12b
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36
Page: https://app.couchers.org/account-settings
User (spoofable): paulpd (429)

### PR DESCRIPTION
**Pull request by lucaslcode**
closes #1402 
fix password change messages
make password reset complete page fullscreen because it was trying to call 'Ping' and causing unauthenticated logout

**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [x] All tests pass
- [x] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Review by darrenvong — COMMENTED** [review]
I got mixed up with this with the change password form once you are logged in! Currently even when you clear your password, it'd still say "your new password has been set...". Do we want to fix that or just leave it for now?

File: app/frontend/src/features/auth/password/CompleteResetPassword.tsx [hunk 1]
```diff
@@ -32,20 +38,24 @@ export default function CompleteResetPassword() {
...
+            {CLICK_LOGIN}
+          </MuiLink>
+        </>
+      ) : error ? (
+        <Alert severity="error">
+          {`${RESET_PASSWORD_ERROR}${error.message}`}
+        </Alert>
+      ) : (
+        ""
+      )}
```
Comment (darrenvong) [inline app/frontend/src/features/auth/password/CompleteResetPassword.tsx:58]:
```suggestion
      ) : null}
```

**Review by darrenvong — COMMENTED** [review]
Last minor thing!

File: app/frontend/src/features/auth/password/CompleteResetPassword.tsx (hunk 1 above)
Comment (lucaslcode) [reply app/frontend/src/features/auth/password/CompleteResetPassword.tsx:58]: From memory, I originally had this but got type errors

Comment (lucaslcode) [reply app/frontend/src/features/auth/password/CompleteResetPassword.tsx:58]: "fixed in 5.0" https://github.com/mui-org/material-ui/issues/21711

**Comment by lucaslcode** [comment]
CI failed on protos but tests are passing for me

**Review by darrenvong — APPROVED** [review]

**Pull request merged**